cosmos/kaiyo.json

```diff
@@ -52,6 +52,12 @@
       "coinDecimals": 6,
       "coinGeckoId": "aqualibre",
       "coinImageUrl": "https://raw.githubusercontent.com/chainapsis/keplr-chain-registry/main/images/kaiyo/factory/kujira1xe0awk5planmtsmjel5xtx2hzhqdw5p8z66yqd/AQLA.png"
+    },
+    {
+      "coinDenom": "NAMI",
+      "coinMinimalDenom": "factory/kujira13x2l25mpkhwnwcwdzzd34cr8fyht9jlj7xu9g4uffe36g3fmln8qkvm3qn/unami",
+      "coinDecimals": 6,
+      "coinImageUrl": "https://raw.githubusercontent.com/chainapsis/keplr-chain-registry/main/images/kaiyo/factory/kujira13x2l25mpkhwnwcwdzzd34cr8fyht9jlj7xu9g4uffe36g3fmln8qkvm3qn/NAMI.png"
```

Comment:
Hi. Could you please update the name of the image file to `unami.png`, exactly following `coinMinimalDenom`?

Reply:
Hi thanks for the feedback. I have changed the file name and path.